src/test/java/de/uhd/ifi/se/decision/management/jira/extraction/versioncontrol/TestGitRepositoryFSManager.java

```diff
@@ -1,32 +1,41 @@
 package de.uhd.ifi.se.decision.management.jira.extraction.versioncontrol;
 
-import org.junit.Before;
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
 
-import javax.xml.bind.DatatypeConverter;
 import java.io.File;
 import java.io.IOException;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
-import static org.junit.Assert.*;
+import javax.xml.bind.DatatypeConverter;
 
-public class TestGitRepositoryFSManager {
+import org.junit.Before;
+import org.junit.Test;
 
+public class TestGitRepositoryFSManager {
 
-	private String baseProjectUriDir;
-	private String baseProjectUriDefaultDir;
-	private String baseProjectUriTempDir;
-	private String expectedBaseProjectUriBranchDir;
+	private List<String> baseProjectUriDir;
+	private List<String> baseProjectUriDefaultDir;
+	private List<String> baseProjectUriTempDir;
+	private List<String> expectedBaseProjectUriBranchDir;
+	private List<String> repoUris;
+	private Map<String, String> folderForDefaultBranchNames;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'folderForDefaultBranchNames' could be replaced by a local variable.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=4916975)